src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx

```diff
@@ -52,7 +52,30 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
   const [showStageGateModal, setShowStageGateModal] = useState<boolean>(false);
   const [showDeleteModal, setShowDeleteModal] = useState<boolean>(false);
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [tab, setTab] = useState(0);
   const dropdownOpen = Boolean(anchorEl);
+  const tabUrlValues = useMemo(() => ['overview', 'scope', 'changes'], []);
```

Comment:
honestly i feel like we should somehow be able to abstract this tab process to a component because theyre essentially all the same and we use them alot

Reply:
we definitely can, I'd say let's make another ticket for it tho